cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt

```diff
@@ -223,6 +294,15 @@ policy 'azure_license_policy' do
       field "name" do
```

Comment:
Incident output should have both subscription name and subscription id, and they should match accountname and accountid like our other policies to ensure the results are properly scraped. See the incident output section of this policy and try to include everything that would also make sense in this policy, with the same names:

https://raw.githubusercontent.com/flexera-public/policy_templates/master/cost/azure/old_snapshots/azure_delete_old_snapshots.pt